lms/product/plugin/course_copy.py

```diff
@@ -0,0 +1,93 @@
+from typing import Callable, Optional
+
+from lms.models import File
+from lms.services.exceptions import ExternalRequestError
+from lms.services.file import FileService
+
+
+class FileNotFoundInCourse(Exception):
+    pass
+
+
+class CourseCopyFilesHelper:
```

Comment:
Collaborator centered around files. These are all staticfunctions.

I rekcon we'll have soon a `CourseCopyGroupsHelper`